docs/blog/2018-10-26-introducing-gatsby-themes/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Introducing Gatsby Themes
+date: 2018-10-26
+author: Chris Biscardi
+excerpt: "Scaling the production of Gatsby sites"
+tags: ["themes", "architecture"]
+---
+
+Gatsby is a powerful platform for building marketing sites, blogs,
+ecommerce frontends, and more. We can source data from static files
+and any number of content management systems. We can process images,
+add support for our favorite styling technique, transform markdown,
+and just about anything else you can imagine.
+
+At its core, a Gatsby site is a combination of functionality centered
+around a single config file, `gatsby-config.js`. This config file
+controls an assortment of site metadata, data type mapping, and most
+importantly, plugins. Plugins contain large amounts of customizable
+functionality for turning markdown into pages, processing components
+into documentation, and even processing images.
+
+## Scaling Gatsby
+
+Creating one Gatsby site is great but we had to start from scratch and
+make all the decisions ourselves. Those decisions are encoded into our
+`gatsby-config`. The next time we have to create a similar site for a
+new product we have to start from scratch and remake all of the
+decisions we made for our first site. Wouldn't it be great if we could
+re-use our config across projects? That's where starters come
+in.
+
+### Improving Reusability with Starters
+
+One way to create more sites with similar functionality faster is to
+use starters. Starters are basically whole Gatsby sites that can be
+scaffolded through the gatsby CLI. This helps you start your project
+by cloning the boilerplate, installing dependencies, and clearing Git
+history. The community around Gatsby has built a lot of different
+starters for various use cases including blogging, working with
+material design, and documentation.
+
+The problem with starters is that they're one-offs. Starters are
```

Comment:
This is where it starts to get interesting for me. I think the stuff before this could be condensed; do readers need a definition of starters?

Reply:
We could maybe just link to starters for those who aren't sure what this means to avoid distracting from the point of the post?

Reply:
While we're mostly reaching out to people who already know what themes are and why they'd want them with this blog post, I included this section to point out how starters are one step up on "the reusability chain": more reuse than a regular site but less than themes. "How are starters and themes different" is also a question I see as going to come up a lot, so defining starters next to themes helps people contrast the approaches side-by-side.

Reply:
Ok, I can see the purpose then! Cool!